database/migrations/2022_07_07_181045_add_is2faenabled_collumn_to_users_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table("users", function (Blueprint $table): void {
+            $table->boolean("is_two_fa_enable")->default(false);
```

Comment:
In `[database/migrations/2022_07_07_175410_create_user_2fa_codes_table.php](https://github.com/blumilksoftware/meetup/pull/123/files#diff-145ca360beb4f1c49c8d2f834a62e262d5beaeca6f03f92b2b503af13361e4c8)` there is `2fa` and here it's `two_fa`. Please unify it.

Reply:
Also: `has_2fa_enabled` is more precise, I think.

Reply:
done